cli/cmd/alpha_stat.go

```diff
@@ -84,7 +85,16 @@ Examples:
 				return err
 			}
 
-			target, err := util.BuildResource(options.namespace, args[0])
+			namespace := options.namespace
+			if options.allNamespaces {
+				namespace = ""
```

Comment:
I think it makes sense to use `corev1.NamespaceAll` here, unless you are against importing that